tests/testsuite.src/used_binaries.at

```diff
@@ -387,11 +387,12 @@ AT_DATA([prog.cob], [
            EXIT PROGRAM.
 ])
 
-AT_CHECK([TMPDIR="" TMP="notthere" TEMP="" $COMPILE prog.cob], [0], [],
```

Comment:
Please add a short comment above why we only use `COMPILE_ONLY` (maybe move that to the more complicated testcase to postpone, we actually do want to check that the C compile and link phase does work, here).

Reply:
> we actually do want to check that the C compile and link phase does work, here

That's why after this "compile-only" check, we also perform a regular compile (check line 393).